Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs

```diff
@@ -458,14 +458,14 @@ internal BrowserCommandResult<bool> OpenSubArea(string area, string subarea, int
             return Execute(GetOptions("Open Sub Area"), driver =>
             {
                 //If the subarea is already in the left hand nav, click it
-                var success = TryOpenSubArea(subarea);
+                var success = TryOpenSubArea(driver, subarea);
                 if (!success)
                 {
                     success = TryOpenArea(area);
                     if (!success)
                         throw new InvalidOperationException($"Area with the name '{area}' not found. ");
 
-                    success = TryOpenSubArea(subarea);
+                    success = TryOpenSubArea(driver, subarea);

```

Comment:
TryOpenSubArea is a private auxiliar methods, is not needed to create a Command for that, pass the driver.